src/ripple/conditions/impl/utils.h

```diff
@@ -0,0 +1,368 @@
+//------------------------------------------------------------------------------
```

Comment:
These functions are exercised indirectly, since they're used when generating the string or blob versions of conditions or fulfillments.